prov/shm/src/smr_rma.c

```diff
@@ -109,6 +103,8 @@ static ssize_t smr_generic_rma(struct smr_ep *ep, const struct iovec *iov,
 	ssize_t ret = 0;
 	size_t total_len;
 	bool use_ipc;
+	struct smr_cmd_entry *cmd;
```

Comment:
In some places of the code, you changed entries to be called ce and others you left as cmd. Let's align all of them for readability. I'm fine with ce, cmd_entry, or entry. I'll leave that up to you.